src/main/java/com/zipcodewilmington/looplabs/IntegerDuplicateDeleter.java

```diff
@@ -5,4 +5,73 @@
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class IntegerDuplicateDeleter extends DuplicateDeleter<Integer> {
+
+    public IntegerDuplicateDeleter(Integer[] intArray) {
+        super(intArray);
+    }
+
+    @Override
+    public Integer[] removeDuplicates(int maxNumberOfDuplications) {
+
+        StringBuilder taco = new StringBuilder();
+
+        for (Integer loopingTaco : array) {
+            int count = 0;
+
+            for (int i = 0; i < array.length; i++) {
+                Integer comparingTaco = array[i];
+                if (loopingTaco == comparingTaco)
+                    count++;
+            }
+
+            if (count < maxNumberOfDuplications) {
+                taco.append(loopingTaco);
+                taco.append(" ");
+            }
+        }
+
+        String[] firstTry = taco.toString().split(" ");
```

Comment:
doesn't look like you use firstTry anywhere else. This is wasted and also a terrible variable name